packages/components/button/src/button.vue

```diff
@@ -85,7 +85,9 @@ const shouldAddSpace = computed(() => {
 })
 
 // calculate hover & active color by color
-const typeColor = computed(() => useCssVar(`--el-color-${props.type}`).value)
+const typeColor = computed(
+  () => useCssVar(`--el-color-${props.type || _type.value}`).value
```

Comment:
**Question:** since`_type.value` is declared:

```ts
const _type = computed(() => props.type || buttonGroupContext?.type || '')
```
Don't you think that only `_type.value` is enough?

Reply:
Yes, `_type.value` is enough. Thanks for the review, i'll update that solution.